Backend/ManagerLayer/Login/LoginManager.cs

```diff
@@ -26,11 +26,11 @@ public LoginManager()
             _tokenService = new TokenService();
         }
 
-        public bool LoginCheckUserExists(string email)
+        public bool LoginCheckUserExists(LoginRequest request)
         {
             using (var _db = new DatabaseContext())
             {
```

Comment:
DB access should be put i the route layer for transactions

Reply:
if I do that a lot of the logic will move in the controller and make the method will be bigger, wouldn't it? @alfredovargasjr